### PR TITLE
move status-im from utils

### PR DESCRIPTION
**Pull request by flexsurfer**
move status-im from utils

### CODE REVIEW

File: src/status_im/subs/contact.cljs [hunk 1]
```diff
@@ -40,14 +41,20 @@
...
+                                     :public-key
+                                     public-key
+                                     :image-name
+                                     image-name
                                      ; We pass the clock so that we reload the
                                      ; image if the image is updated
-                                     :clock          clock
-                                     :theme          theme
-                                     :override-ring? (when ens-name false)})]
+                                     :clock
```
Comment (flexsurfer) [inline src/status_im/subs/contact.cljs:52]: have no idea why it was moved next line

File: scripts/lint/re-frame-in-quo-components.sh [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
+fi
+
+INVALID_CHANGES3=$(grep -E -r '(status-im\.)' --include '*.cljs' --include '*.clj' './src/react_native')
+
+if test -n "$INVALID_CHANGES3"; then
+    echo "WARNING: status-im are not allowed in react-native package"
+    echo ''
+    echo "$INVALID_CHANGES3"
+    exit 1
+fi
```
Comment (siddarthkay) [inline scripts/lint/re-frame-in-quo-components.sh:28]: A better way could be : 
```bash

check_invalid_changes() {
    local pattern=$1
    local path=$2
    local message=$3

    local invalid_changes=$(grep -E -r "$pattern" --include '*.cljs' --include '*.clj' "$path")

    if test -n "$invalid_changes"; then
        echo "WARNING: $message"
        echo ''
        echo "$invalid_changes"
        exit 1
    fi
}


check_invalid_changes '(re-frame/dispatch|rf/dispatch|re-frame/subscribe|rf/subscribe|rf/sub|<sub|>evt|status-im\.)' './src/quo' 're-frame, status-im are not allowed in quo components'
check_invalid_changes '(status-im\.)' './src/utils' 'status-im are not allowed in utils package'
check_invalid_changes '(status-im\.)' './src/react_native' 'status-im are not allowed in react-native package'
```

Comment (flexsurfer) [reply scripts/lint/re-frame-in-quo-components.sh:28]: probably :) I'd leave it to you then

**Review by siddarthkay — APPROVED** [review]
Changes look good, but should be smoke tested once since we're touching UI components...

File: scripts/lint/re-frame-in-quo-components.sh (hunk 2 above)
Comment (siddarthkay) [reply scripts/lint/re-frame-in-quo-components.sh:28]: Thanks, I'll improve this in a follow up :)

**Review by Parveshdhull — APPROVED** [review]

**Comment by flexsurfer** [comment]
@status-im/mobile-qa could someone please take a quick look when have some time, thank you

**Comment by VolodLytvynenko** [comment]
Hey, @flexsurfer. Thank you for your work. I will check this PR. Any known impacted areas?

**Comment by flexsurfer** [comment]
thanks @VolodLytvynenko media server, so images for contacts etc

**Comment by VolodLytvynenko** [comment]
## ISSUE 1: The avatar image is not shown after setup

### Steps to Reproduce:
1. Set an image for the avatar during or after onboarding
2. Navigate to various pages where the avatar should be displayed.

### Actual Result:
The avatar image does not appear on any of the pages where it is expected to be shown.

image
image


### Expected Result:
 the avatar image to be displayed correctly on all pages where the avatar is intended to appear.

### Devices:
- iPhone 11 Pro max, IOS q6
- Pixel 7a, Android 14

### Logs:
[Status-debug-logs.zip](https://github.com/status-im/status-mobile/files/13727804/Status-debug-logs.zip)

**Comment by flexsurfer** [comment]
thanks @VolodLytvynenko fixed

**Comment by VolodLytvynenko** [comment]
@flexsurfer issue 1 is fixed. One more issue is found

## ISSUE 2: User avatar displays as [x] when Image is not set

### Steps to Reproduce:
1. DON'T set image for the avatar during or after onboarding
2. Navigate to various pages where the avatar should be displayed.

### Actual result:
Instead of displaying an avatar placeholder, the [x] symbol is shown in place of the avatar
image
image


### Expected result:
Default avatar placeholder  is shown
image

### Devices:
- iPhone 11 Pro max, IOS q6
- Pixel 7a, Android 14

**Review by yqrashawn — APPROVED** [review]

**Comment by flexsurfer** [comment]
sorry @VolodLytvynenko should be fixed now

**Comment by qoqobolo** [comment]
Thanks for the fixes @flexsurfer, PR is ready to be merged.

**Pull request merged**